dataclients/kubernetes/testdata/routegroups/convert/failing-predicate.log

```diff
@@ -1 +1 @@
-\[routegroup\] parse failed after token Header, last route id: Header, position 6: syntax error
```

Comment:
This shows another problem of previous ParsePredicates implementation: it confused `Header("Foo", "Bar") -> <shunt>` to have route id of `Header`

Reply:
https://github.com/zalando/skipper/pull/2875